graph/articles/collections.md

```diff
@@ -316,3 +316,151 @@ Content-Type: application/json
    "value": []
 }
 ```
+
+## 11. Collections of structural types (complex types or primitive types)
```

Comment:
the second option (changing the property to a nav prop) doesn't really work for collections of primitive types, right?